<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>258</width>
      <height>290</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QListWidget" name="tasksListWidget">
       <property name="styleSheet">
        <string notr="true">background-color:OldLace</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="detailsTaskButton">
       <property name="text">
        <string>Подробности задачи</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addTaskButton">
       <property name="text">
        <string>Добавить задачу</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteTaskButton">
       <property name="text">
        <string>Удалить задачу</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
